src/extension.ts

```diff
@@ -1731,9 +1736,34 @@ async function deleteContextKubernetes(explorerNode: explorer.KubernetesObject)
     }
 }
 
-async function useNamespaceKubernetes(explorerNode: explorer.KubernetesObject) {
-    if (await kubectlUtils.switchNamespace(kubectl, explorerNode.id)) {
-        refreshExplorer();
+async function useNamespaceKubernetes(explorerNode?: explorer.KubernetesObject) {
+    if (explorerNode) {
+        if (await kubectlUtils.switchNamespace(kubectl, explorerNode.id)) {
+            refreshExplorer();
+        }
+    } else {
+        promptKindName([kuberesources.allKinds.namespace], 'get', undefined, (resource) => {
```

Comment:
This will result in a prompt of "What resource do you want to get?"; suggest we should aim for e.g. "What namespace do you want to use?" or "Enter the namespace to switch to."